### PR TITLE
Restructure project (#46)

### PR DESCRIPTION
**Pull request by yHoonyK**
* Create nugu-ios.xcodeproj
* Update version in podspec

### CODE REVIEW

**Comment by yHoonyK** [comment]
#### Remain works
1. Setting travis
2. Update Readme 
3. Support Carthage
4. Support SPM

File: .jazzy.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# nugu-ios document generator jazzy settings
+
+copyright: Copyright 2019 SK Telecom Co., Ltd.
```
Comment (nugu-dev-sdk) [inline .jazzy.yaml]: author 수정해주세요.

File: .swiftlint.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - switch_case_on_newline
+  - force_unwrapping
+  - conditional_returns_on_newline
+  - closure_spacing
+  - implicitly_unwrapped_optional
+  - sorted_imports
+  - valid_docs
+  - nslocalizedstring_require_bundle
+
+included:
```
Comment (nugu-dev-sdk) [inline .swiftlint.yml]: included 확인해주세요.

File: .jazzy.yaml (hunk 1 above)
Comment (yHoonyK) [reply .jazzy.yaml]: jazzy.yaml을 잠시 삭제했습니다.

**Pull request merged**